app/services/api/teachers/school_transfers/query.rb

```diff
@@ -74,10 +74,31 @@ def where_lead_provider_is(lead_provider_id)
         )
     end
 
-    def where_updated_since(updated_since)
+    def where_updated_since(updated_since, lead_provider_id)
       return if ignore?(filter: updated_since)
 
-      @scope = scope.where(api_updated_at: updated_since..)
+      # Note that this will include the first training period of a teacher and the last
+      # training period even if they have completed training (which are not classed as transfers).
+      # Excluding these would be very complicated; this is close enough for now and an improvement
+      # over what ECF currently provides.
```

Comment:
Is it worth highlighting what the impact is on the User here?

I'm assuming this means that the API _might_ return participant transfers that haven't actually been updated since the given date if an excluded training period has been.